packages/connectivity/src/scp-cf/destination/destination-accessor-provider-subscriber-lookup.spec.ts

```diff
@@ -326,18 +330,20 @@ describe('JWT type and selection strategies', () => {
 });
 
 describe('call getAllDestinations with and without subscriber token', () => {
-  beforeEach(() => {
+  beforeEach(async () => {
     mockServiceBindings();
     mockVerifyJwt();
     mockServiceToken();
+    await destinationCache.clear();
+    await destinationServiceCache.clear();
   });
 
   afterEach(() => {
     nock.cleanAll();
     jest.clearAllMocks();
   });
 
-  it('should fetch all subscriber destinations', async () => {
+  it('should fetch all subscriber destinations and cache destinations', async () => {
```

Comment:
Added these `caching` tests here as `getAllDestinationsFromDestinationService()` doesn't allow passing a destination service url, and I did not find an easy way to test this function with the existing setup in `destination-service-cache.spec.ts`

Reply:
Done.